cli/command/config/cmd.go

```diff
@@ -27,3 +27,18 @@ func NewConfigCommand(dockerCli command.Cli) *cobra.Command {
 	)
 	return cmd
 }
+
+// CompletionFn offers completion for swarm secrets
+func CompletionFn(dockerCli command.Cli) command.CompletionFn {
```

Comment:
One thing; so it looks like all completion functions (with the exception of `image.CompletionFn()` and `container.CompletionFn()`) are only used within the package itself;

<img width="727" alt="Screenshot 2022-05-02 at 16 29 41" src="https://user-images.githubusercontent.com/1804568/166251717-02604479-5287-40db-bd6a-24dd947babf3.png">

I generally like to keep things un-exported, unless needed; perhaps we should 

- un-export all of them (except for the `image` and `container` ones)
- should we have more descriptive names (for the exported ones at least)? (e.g., `CompleteNames()` ?); mostly wondering if there will be more "specialized" completion functions (per my earlier comments w.r.t. "in specific states" etc), in which case a more descriptive name would help.